README.md

```diff
@@ -65,7 +65,7 @@ The environment variables are as follows:
 * DEV_DB_CONN_STRING: Used to connect to the dev database
 * VITE_VUE_API_BASE_URL: The base URL for the API
 * VITE_VUE_APP_BASE_URL: The base URL for the UI
-* SECRET_KEY: Used to sign cookies
```

Comment:
on my end, the SECRET_KEY looks like an API key that doesn't seem to refresh. Is that accurate? Were the two concepts somehow mixed?

Reply:
Technically, a SECRET_KEY can be anything. It's used as a key for the encryption and decryption of values (in this case, to ensure cookies aren't being forged). I did rename the environment variable to `FLASK_APP_SECRET_KEY` to be more precise. The `JWT_SECRET_KEY` servers a similar function, but in this case to encrypt the JWTs.

Reply:
I see. The root of my question: is an API key no longer required to run the app locally? It looks like the envars bit of the README is still a bit out of date

Reply:
nice, these instructions are helpful. Agreed that the swagger docs should tell people _how_ the `...test-user...` endpoint works, but we need to tell people _that they need to use it._